Explicitly distinguish Callbacks from Events (#827)
ietf-tapswg/api-drafts: draft-ietf-taps-interface.md

Security decisions, especially pertaining to trust, are not static. Once configured,
parameters may also be supplied during connection establishment. These are best
handled as client-provided callbacks. 

> Callbacks block the progress of a connection, which distinguishes them from other Events in the transport system, while implementations may choose to realize them analogous.

Reviewer: ```suggestion
Callbacks block the progress of a connection, which distinguishes them from other Events in the transport system. How callbacks and events are implemented is specific to each implementation.
```

> Callbacks block the progress of a connection, which distinguishes them from other Events in the transport system. How callbacks and events are implemented is specific to each implementation.

Reviewer 2: ```suggestion
Callbacks block the progress of a Connection's establishment, which distinguishes them from other Events in the transport system. How callbacks and events are implemented is specific to each implementation.
```

Author: I would prefer to leave the connection on lower case (as I am not sure that the connection object we refer to in upper case has/makes progress, but something in the underlaying transport system).

 ```suggestion
Callbacks block the progress of the connection establishment, which distinguishes them from other Events in the transport system. How callbacks and events are implemented is specific to each implementation.
```

Reviewer 2: ok, this makes sense to me
